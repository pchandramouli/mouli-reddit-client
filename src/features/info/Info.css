
.posts .info {
  display: flex;
  justify-content: space-between;
  color: #999;
  max-width: 960px;
  min-width: 640px;
  border: 1px solid #fff;
  border-radius: 5px;
  background-color: #fff;
  padding: 0.5rem 1rem;
  box-shadow: silver 2px 2px;
}

.posts .info div {
  display: inline-flex;
}

.posts .info img {
  width: 16px;
  height: 16px;
}