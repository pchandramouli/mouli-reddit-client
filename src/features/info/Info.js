import upvoteImg from "../../app/images/up-long-solid.svg";
import downvoteImg from "../../app/images/down-long-solid.svg";
import './Info.css';

export const Info = ({upvotes, downvotes}) => {
  return (
    <div className="info">
      <div>
        <span><img src={upvoteImg}/>{upvotes}</span>
      </div>
      <div>
        <img src={downvoteImg}/><span>{downvotes}</span>
      </div>
    </div>
  );
}