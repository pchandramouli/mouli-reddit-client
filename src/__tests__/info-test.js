import renderer from 'react-test-renderer';
import { Info } from '../features/info/Info';

describe('Info',() => {
  it('render properly', () => {
    const component = renderer.create(<Info upvotes="10" downvotes="2" className="votes"></Info>);
    expect(component.toJSON()).toMatchSnapshot();
  })
})