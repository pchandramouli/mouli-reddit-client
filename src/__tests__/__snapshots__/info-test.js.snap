// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info render properly 1`] = `null`;
